index.js

```diff
@@ -1 +1,89 @@
-// Do work!
\ No newline at end of file
+/* eslint-disable indent */
+// // Make a password function
+// function validatePassword(password) {
+// let status = []
+
+//     if () {
+//         let status = true;
+//     }
+
+
+
+// // Success! 
+
+// // Not long enough, needs >= 8 chars
+//     if (password.length < 8) {
+//         let status = false;
+//     }
+// // Missing a lower case letter
+//     if (password.case) {
+
+//     }
+// // Missing an upper case letter
+
+// // Missing a numeric value
+
+// // Missing a special character
+
+// }
+
+
+
+
+
+function validatePassword(password) {
+    const validLength = (password.length >= 8)
+
+    const validChars = pushCharacters(password)
+
+    if (validLength && validChars) {
+        return true
+    }
+
+    return false
+}
+
+function pushCharacters(password) {
+    let specChars = '`~!@#$%^&*()_+'
+
+    let charStatus = {
+        'uppercase': false,
+        'lowercase': false,
+        'numerical': false,
+        'special': false
+    }
+
+
+    let passwordArray = password.split('')
+
+    passwordArray.forEach((character) => {
+        switch (true) {
```

Comment:
This is the most original use of a `switch` statement I have seen. 
Normally we have a variable and then fixed values for the case statements but you are using some evaluation I have not seen done in a case before.  If it works as it appears to I need to learn more about using switch like this.
I need to try this out in few examples. You may be onto something.